cores/cv32e40p/user_manual/source/control_status_registers.rst

```diff
@@ -451,3 +466,141 @@ Reset Value: 0x0000_0000
 +-------------+
 
 Scratch register that can be used by implementations that need it.
+
+
+Trigger Select Register (tselect)
+---------------------------------
+
+CSR Address: 0x7A0
+
+Reset Value: 0x0000_0000
+
+Accessible in Debug Mode or M-Mode when trigger support is enabled (using the DbgTriggerEn parameter).
+
+CV32E40P implements a single trigger, therefore this register will always read as zero
+
+
+Trigger Data Register 1 (tdata1)
+--------------------------------
+
+CSR Address: 0x7A1
+
+Reset Value: 0x2800_1000
+
+Accessible in Debug Mode or M-Mode when trigger support is enabled (using the DbgTriggerEn parameter).
+Since native triggers are not supported, writes to this register from M-Mode will be ignored.
+
+CV32E40P only implements one type of trigger, Match Control. Most fields of this register will read as a fixed value to reflect the single mode that is supported, in particular, instruction address match as described in the Debug Specification 0.13.2 section 5.2.2 & 5.2.9.
+
+
++-------+------+------------------------------------------------------------------+
+| Bit#  | R/W  | Description                                                      |
++-------+------+------------------------------------------------------------------+
```

Comment:
In RST tables, the line under the header is indidcated using "=".   So line 498 is should be:
`+=======+=======+===============================================+`
The "+" characters needs to line up with the "|" characters on the preceding line.